### ISSUE TITLE
Deleted items go to trash

### ISSUE THREAD

**Issue by alexpdraper** [alexpdraper/reading-list#71]
If we add a trash bin, we can send “deleted” items to the trash, and then only delete them after a set period of time (30 days or whatever is best). It would also allow users to undo accidental deletions.

**Comment by hklene** [comment]
This might actually play nice with #50 to inform the user of what has been done. On the other hand, with #54 the trash would also be a bookmark folder, so deletion will still keep the link around for a bit longer. Then a short turnaround time like only a few days will be nice, to really get rid of items. But the shorter the period, the more important the exact countdown becomes. Eventually you don't want your work computer to empty the trash on Monday morning after the weekend, even though, you set the countdown to 3 days cause on Thursday, you don't need the old stuff from Monday anymore.

**Comment by alexpdraper** [comment]
I agree this would play nice with #50. I think I’d make the default time limit longer than three days, instead of worrying about not deleting things on weekends.
